src/components/map/MapService.js

```diff
@@ -771,6 +772,20 @@ goog.require('ga_urlutils_service');
               .replace('{Lang}', lang);
         };
 
+        // Load layers for a given topic and language. Return a promise.
+        var loadLayersConfig = function(lang) {
+          var url = getLayersConfigUrl(lang);
+          return $http.get(url).then(function(response) {
+            var isLabelsOnly = angular.isDefined(layers);
+            layers = response.data;
```

Comment:
Maybe we can even get rid of this `isLabelsOnly` parameter (artefact of per topic layers config loading). This parameter is false on for the first time, after that, it's always true.

So by using the gaTopic event instead of this variable, I think we can init the other dependant components correctly. But it's maybe not as trivial as it sounds.